### DISCUSSION TITLE
Include support for docker port mapping.

### DISCUSSION THREAD

**Discussion by ajjswift** [pelican-dev/panel#112] [Ideas]
Would be really amazing to include support for port binding on containers, so that games that require certain ports (such as FiveM) can be hosted more easily and without a dedicated IP address.

For the example of FiveM, it requires 30110, 30120, 40120, but with port mapping we could have 30110 bind to 25478, 30120 to 54678, and 40120 to 34576.

Just an example, but would be a cool improvement to the networking tab. (Maybe under an advanced toggle as to not confuse people).

**Comment by parkervcp** [comment]
We already replace the ports for fivem when it starts and they are not required to be on those ports. There should be no reason to need to port map those.

As the person who wrote the egg, and verified in as many ways as I could, we shouldn't need to do that.

**Reply by parkervcp** [reply]
As for other games and servers, I haven't seen one with an issue not having the external port be a specific one.
